package.json

```diff
@@ -21,10 +21,13 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "~12.2.0",
+    "@angular/cdk": "^12.2.12",
     "@angular/common": "~12.2.0",
     "@angular/compiler": "~12.2.0",
     "@angular/core": "~12.2.0",
-    "@angular/forms": "~12.2.0",
+    "@angular/flex-layout": "^12.0.0-beta.35",
+    "@angular/forms": "^12.2.12",
+    "@angular/material": "^12.2.12",
```

Comment:
use `npm` to remove angular material dependencies 

https://docs.npmjs.com/uninstalling-packages-and-dependencies